docs/docsite/rst/dev_guide/testing/sanity/no-tests-as-filters.rst

```diff
@@ -1,10 +0,0 @@
-Sanity Tests » no-tests-as-filters
```

Comment:
I'd keep this file around even though the test has been deleted. That way links from old test runs will continue to work. I've done this for several other tests which no longer exist.

Reply:
Ok, I can do that.  I'll just have to indicate it is an orphan